Mixpanel/MixpanelInstance.swift

```diff
@@ -85,12 +85,9 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate {
     /// The base URL used for Mixpanel API requests.
     /// Useful if you need to proxy Mixpanel requests. Defaults to
     /// https://api.mixpanel.com.
-    open var serverURL: String {
-        set {
-            BasePath.MixpanelAPI = newValue
-        }
-        get {
-            return BasePath.MixpanelAPI
+    open var serverURL = BasePath.DefaultMixpanelAPI {
```

Comment:
Right, initially setting the variable does not call `didSet`.  From apple docs:

> willSet and didSet observers are not called when a property is first initialized. They are only called when the property’s value is set outside of an initialization context.

Instead, I actually have `getServerURL()` (the function which uses `BasePath.namedBasePaths`) fall back to `BasePath. DefaultMixpanelAPI` if we don't have a `serverURL` set.

Would you like me to leave a comment?